src/fetch/solver_info.py

```diff
@@ -0,0 +1,64 @@
+"""Functionality for solver information."""
+
+import numpy as np
+import pandas as pd
+from pandas import DataFrame
+
+from src.config import AccountingConfig
+from src.logger import set_log
+
+log = set_log(__name__)
+
+REWARD_TARGETS_COLUMNS = ["solver", "reward_target", "pool_address", "solver_name"]
+SERVICE_FEES_COLUMNS = ["solver", "service_fee"]
+
+SOLVER_INFO_COLUMNS = [
+    "solver",
+    "solver_name",
+    "reward_target",
+    "buffer_accounting_target",
+    "service_fee",
+]
+
+
+def compute_solver_info(
+    reward_targets: DataFrame,
+    service_fees: DataFrame,
+    config: AccountingConfig,
+) -> DataFrame:
+    """Compute solver information"""
```

Comment:
A comment about the input, and specifically about the `service_fee` column (is it a boolean or a fraction), would help